pyproject.toml

```diff
@@ -38,23 +38,21 @@ authors = [
 ]
 
 dependencies = [
-  "astropy >= 5.3.0",
-  "matplotlib >= 3.5.0",
-  "numpy >=1.23.5",
-  "scikit-image >= 0.19.0",
-  # !=1.10.0 due to https://github.com/scipy/scipy/issues/17718
-  "scipy >= 1.9.0 , != 1.10.0",
-  "sunpy[map] >= 5.0.0",
+  "astropy >= 6.0.0",
+  "matplotlib >= 3.7.0",
+  "numpy >=1.24.0",
+  "scikit-image >= 0.21.0",
+  "scipy >= 1.11.1",
+  "sunpy[map] >= 5.1.0",
 ]
 
 [project.optional-dependencies]
 dev = [
-  "xrtpy[tests,docs]",
-  "nox >= 2022.8.7",
+  "nox >= 2024.10.9",
 ]
 tests = [
-  "pytest >= 8.0.0",
-  "pytest-astropy",
+  "pytest >= 8.3.1",
+  "pytest-astropy >= 0.11.0",
```

Comment:
Setting the lowest allowed version for all packages makes it more practical to use the `lowest-direct` resolution strategy of `uv`. This strategy finds an environment with the lowest allowed versions of _direct_ dependencies, and preferably the newest allowed versions of _indirect_ dependencies.